js/fedmenu.js

```diff
@@ -108,10 +113,52 @@ var fedmenu = function(options) { $(document).ready(function() {
         });
         if (found) {
             html = html + "</ul></div>";
+            if ($('#fedmenu-package-content').length == 0) {
+                $('body', c).append('<div id="fedmenu-package-content" class="fedmenu-content"></div>');
+                $('#fedmenu-package-content', c).append("<span class='fedmenu-exit'>&#x274C;</span>");
+                $('#fedmenu-package-content', c).append("<h1>View the " + o.package + " package elsewhere</h1>");
+            }
             $("#fedmenu-package-content", c).append(html);
         }
     };
 
+    // A handy lookup for those functions we just defined.
+    var content_makers = {
+        'main': make_main_content_html,
+        'user': make_user_content_html,
+        'package': make_package_content_html,
+    };
+
+    // Figure out the current site that we're on, if possible, and return the
+    // data we have on it from the json we loaded.
+    var get_current_site = function() {
+        var found = null;
+        var ours = window.location.toString();
+        ours = ours.slice(ours.indexOf('://') + 3)
+        $.each(master_data, function(i, node) {
+            $.each(node.children, function(j, leaf) {
+                var theirs = leaf.data.url;
+                theirs = theirs.slice(theirs.indexOf('://') + 3)
+                if (theirs.indexOf(ours) === 0) found = leaf;
+            })
+        });
+        return found;
+    }
+
+    // Try to construct a little footer for the menus.
+    var add_footer_links = function() {
+        var site = get_current_site();
+        var content = "";
+        if (site != null && site.data.bugs_url != undefined && site.data.source_url != undefined) {
```

Comment:
Should we allow to specify only one of the two? (ie: specify a bugs_url but not the source_url or vice-versa)